templates.go

```diff
@@ -10,8 +10,16 @@ import (
 	"strings"
 )
 
+var funcMap = template.FuncMap{
+	"escape": func(text string) template.HTML {
+		text = strings.Replace(text, " ", "` ", -1)
+		text = strings.Replace(text, "\n", "`r`n", -1)
+		return template.HTML(text)
+	},
+}
+
 // compile all templates and cache them
-var templates = template.Must(template.ParseGlob("./templates/*.html"))
+var templates = template.Must(template.New("").Funcs(funcMap).ParseGlob("./templates/*.html"))
```

Comment:
This is doing something Windows specific to all templates right? - could you explain the intention here? Is it just to prettify the output?